flutter_app/lib/modules/return_to_launch.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:dart_mavlink/dialects/common.dart';
+import 'package:dart_mavlink/mavlink.dart';
+import 'package:imacs/modules/mavlink_communication.dart';
+import 'dart:developer';
+
+const String moduleName = "Return To Launch";
+
+class ReturnToLaunch {
+  final MavlinkCommunication comm;
+  final MissionItem returnToLaunchConstructor;
+
+  // Requires both the constructor (Mission Item) and comm
+  ReturnToLaunch({required this.comm, required this.returnToLaunchConstructor});
```

Comment:
You can call the command constructor within the module (look at change_drone_mode for an example)